src/types/widget.ts

```diff
@@ -34,5 +35,9 @@ export interface ChartKitWidget {
     d3: {
         data: ChartKitWidgetData;
         widget: never;
+        splitTooltip?: {
```

Comment:
Are you sure you need it separately? Why not inside ChartKitWidgetData?

Reply:
`ChartKitWidgetData` is about library data only. `splitTooltip` is Chartkit functionality and I think it should be located out of the library data